ui/lib/config-ui/addon/components/messages/tab-page-header.hbs

```diff
@@ -0,0 +1,51 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<PageHeader as |p|>
+  {{#if @breadcrumbs}}
+    <p.top>
+      <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+    </p.top>
+  {{/if}}
+  <p.levelLeft>
+    <Hds::Text::Display @tag="h2" @size="500" class="has-top-margin-xl">
+      Custom messages
+    </Hds::Text::Display>
+  </p.levelLeft>
+</PageHeader>
+
+<div class="tabs-container box is-bottomless is-marginless is-paddingless">
+  <nav class="tabs" aria-label="custom-messages">
+    <ul>
+      <LinkTo
+        class={{if @authenticated "active"}}
+        @route="messages"
+        @query={{hash authenticated=true page=1}}
+        data-test-custom-messages-tab="After user logs in"
+      >
+        After user logs in
+      </LinkTo>
+      <LinkTo
+        class={{unless @authenticated "active"}}
```

Comment:
you might also try using the ember truth-helper not. https://github.com/jmurphyau/ember-truth-helpers. It reads easier than unless.

Reply:
The linter recommends `unless` instead of `if not @authenticated`🫠

Reply:
TIL.